src/skills-builder/skills-builder-modal/select-preferences/CareerInterestSelect.jsx

```diff
@@ -26,10 +27,13 @@ const CareerInterestSelect = () => {
   return (
     <Stack gap={2}>
       <h4>
-        <FormattedMessage {...messages.careerInterestPrompt} />
+        {formatMessage(messages.careerInterestPrompt)}
       </h4>
       <InstantSearch searchClient={searchClient} indexName={getConfig().ALGOLIA_JOBS_INDEX_NAME}>
-        <JobTitleInstantSearch onSelected={handleCareerInterestSelect} />
+        <JobTitleInstantSearch
+          onSelected={handleCareerInterestSelect}
+          placeholder={formatMessage(messages.careerInterestInputPlaceholder)}
```

Comment:
What is the purpose of passing in a static placeholder string as a prop to this component? Wouldn't this just be rendered as part of the component?

Reply:
The `JobTitleInstantSearch` component is used in two places so we needed somewhere to pass a unique placeholder for each instance

Reply:
Ahhhhhh, gotcha. Thanks.